programs/autocrat/src/state/dao.rs

```diff
@@ -25,6 +28,8 @@ pub struct Dao {
 
     pub slots_per_proposal: u64,
 
+    // what's an atom? maybe a comment would be good
```

Comment:
Yeah, its nomenclature I've seen around but I don't think I used it in all cases here. It just means the raw token amount (the big number). I'll change or comment.